src/models/voice_processing/samo_whisper_transcriber_original.py

```
@@ -0,0 +1,613 @@
+                    logger.warning(
+                        "Detected corrupted or missing model files in cache. "
+                        "Clearing cache directory: %s", cache_dir
+                    )
+                    shutil.rmtree(cache_dir)
+                    os.makedirs(cache_dir, exist_ok=True)
+                self.model = whisper.load_model(
+                    self.config.model_size,
+                    device=self.device,
+                    download_root=cache_dir
+                )
+            except (RuntimeError, OSError) as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)